File: Beecrowd Problems/1079.js
/*
Author: Imran Parthib
Date: 14.07.2025 Time: 11:49
*/
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

let inputLines = [];
let testCaseCount = 0;

rl.on('line', (line) => {
    inputLines.push(line.trim());
    if (inputLines.length === 1) {
        testCaseCount = parseInt(inputLines[0]);
    } else if (inputLines.length === testCaseCount + 1) {
        rl.close(); // Stop reading input when all test cases are in
    }
});

rl.on('close', () => {
    for (let i = 1; i <= testCaseCount; i++) {
        const [a, b, c] = inputLines[i].split(' ').map(Number);
        const weightedAvg = (a * 2 + b * 3 + c * 5) / 10;
        console.log(weightedAvg.toFixed(1));
    }
});

File: Beecrowd Problems/1079.cpp
/*
Author: Imran Parthib
Date: 15.07.2025 Time: 12:00
*/
#include <bits/stdc++.h>
using namespace std;

int main()
{
    int t;
    cin >> t;
    float a, b, c;

    while (t--)
    {
        float weightSum = 0;
        cin >> a >> b >> c;
        weightSum = (a * 2 + b * 3 + c * 5) / (2 + 3 + 5);
        cout << fixed << setprecision(1) << weightSum << endl;
    }
    return 0;
}